app/views/candidate_mailer/withdraw_last_application_choice.text.erb

```diff
@@ -3,3 +3,13 @@ Dear <%= @application_form.first_name %>
 You’ve withdrawn your <%= 'application'.pluralize(@withdrawn_course_names.size) %> for <%= @withdrawn_course_names.to_sentence %>.
 
 <%= render 'still_interested_content' %>
+
+<% if @application_form.show_new_reference_flow? %>
+Since you withdrew your application, we’ve contacted these people to say they do not need to give a reference:
```

Comment:
This is in the new copy but I can't see where this happens in the withdraw application service